devtools/cli/src/main/java/io/quarkus/cli/create/IncludeExampleCodeGroup.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.cli.create;
+
+import java.util.Set;
+
+import picocli.CommandLine;
+
+public class IncludeExampleCodeGroup {
+    @CommandLine.Option(names = {
+            "--no-code" }, description = "Include example code or generate an empty project", negatable = true)
```

Comment:
```suggestion
            "--no-code" }, description = "Include starter code provided by extensions or generate an empty project", negatable = true)
```

Reply:
So you understand, we stopped using the term "example code" for extensions provided code, the generated code is not meant to be deleted anymore but to get started from.

We can remove the `--example` option for now (this is meant for a different usecase but let's see if it's requested)

Reply:
ok. Was using old terminology from the CLI/Maven repo as I could see it